tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -5524,171 +5524,145 @@ auto get_concat_nodedef = [](DataType dtype, int num_inputs) -> NodeDef {
+  }
 
   struct TestParams {
     std::vector<std::vector<int>> input_shapes;
-    std::vector<std::vector<CType>> input_values;
+    std::vector<std::vector<int>> input_values;
     int axis;
     std::vector<int> expected_output_dims;
-    std::vector<CType> expected_output;
+    std::vector<int> expected_output;
+    Status conversion_status;
+    Status run_status;
+    bool input_as_weight;
   };
 
-  const std::vector<std::vector<CType>> common_input{
-      InitTestVector<CType>(6),
-      InitTestVector<CType>(6, /*start_value=*/CType(6))};
-  // TODO(hinsu): Use std::vector instead of an array to avoid use of explicit
-  // size.
-  std::vector<TestParams> ok_params = {
+  const std::vector<std::vector<int>> common_input{InitTestVector<int>(6),
+                                                   InitTestVector<int>(6, 6)};
+
+  std::vector<TestParams> params = {
       {
-          /*input_shapes=*/{{1, 2, 3}, {1, 2, 3}},
+          /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
           /*input_values=*/common_input,
           /*axis=*/1,
-          /*expected_output_dims=*/{2, 2, 3},
-          /*expected_output=*/InitTestVector<CType>(12),
+          /*expected_output_dims=*/{1, 2, 2, 3},
+          /*expected_output=*/InitTestVector<int>(12),
       },
       {
-          /*input_shapes=*/{{1, 2, 3}, {1, 2, 3}},
+          /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
           /*input_values=*/common_input,
           /*axis=*/2,
-          /*expected_output_dims=*/{1, 4, 3},
-          /*expected_output=*/InitTestVector<CType>(12),
+          /*expected_output_dims=*/{1, 1, 4, 3},
+          /*expected_output=*/InitTestVector<int>(12),
       },
       {
-          /*input_shapes=*/{{1, 2, 3}, {1, 2, 3}},
+          /*input_shapes=*/{{1, 1, 2, 3}, {1, 1, 2, 3}},
           /*input_values=*/common_input,
           /*axis=*/3,
-          /*expected_output_dims=*/{1, 2, 6},
+          /*expected_output_dims=*/{1, 1, 2, 6},
           /*expected_output=*/
-          {CType(0), CType(1), CType(2), CType(6), CType(7), CType(8), CType(3),
-           CType(4), CType(5), CType(9), CType(10), CType(11)},
+          {0, 1, 2, 6, 7, 8, 3, 4, 5, 9, 10, 11},
       },
       {
-          /*input_shapes=*/{{1}, {2}, {3}, {1}, {1}, {2}},
+          /*input_shapes=*/{{1, 1}, {1, 2}, {1, 3}, {1, 1}, {1, 1}, {1, 2}},
           /*input_values=*/
-          {{CType(1)},
-           {CType(2), CType(3)},
-           {CType(4), CType(5), CType(6)},
-           {CType(7)},
-           {CType(8)},
-           {CType(9), CType(10)}},
+          {{1}, {2, 3}, {4, 5, 6}, {7}, {8}, {9, 10}},
           /*axis=*/1,
-          /*expected_output_dims=*/{10},
+          /*expected_output_dims=*/{1, 10},
           /*expected_output=*/
-          InitTestVector<CType>(10, /*start_value=*/CType(1)),
+          InitTestVector<int>(10, /*start_value=*/1),
       },
-  };
+      {// An input is a weight
```

Comment:
Let's move the comment to a new line.

Reply:
Fixed. Interestingly, to prevent clang format to override my formatting, I need to add a trailing comma before I close the block with }.